### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by dafnevk**
I fixed the table of contents and moved the content around a bit in a more logical order. The content of the technical documentation is updated to match the paper.

### CODE REVIEW

File: docs/installation.rst [hunk 1]
```diff
@@ -1,36 +1,46 @@
 Installation
 ============
 
-Prerequisites:
+Prerequisites: - Python 3.5, 3.6 or 3.7 - pip - Tensorflow 2.0, if pip
```
Comment (florian-huber) [inline docs/installation.rst:4]:
```suggestion
Prerequisites: - Python 3.6, 3.7 or 3.8 - pip - Tensorflow 2.0, if pip
```

File: docs/installation.rst [hunk 2]
```diff
@@ -1,36 +1,46 @@
...
-* Install numpy and scipy through the conda package manager (and not with pip)
-* To install mcfly, run `pip install mcfly` in the cmd prompt.
-* Loading and saving models can give problems on Windows, see https://github.com/NLeSC/mcfly-tutorial/issues/17
+~~~~~~~~~~~~~~~~~~~~~
+
+When installing on Windows, there are a few things to take into
+consideration. The preferred (in other words: easiest) way to install
+Keras and mcfly is as follows: \* Use
+`Anaconda <https://www.continuum.io/downloads>`__ \* Use Python 3.x,
+because tensorflow is not available on Windows for Python 2.7 \* Install
```
Comment (florian-huber) [inline docs/installation.rst:27]: This comment on Python 2.7 seems outdated... shouldn't we remove it?

Comment (dafnevk) [reply docs/installation.rst:27]: yes, I copied this from the README so I will remove it there as well.

**Comment by cwmeijer** [comment]
I only reviewed the 2 commits after Florian's commit. They are simple and clearly improvements. If you are ready, you can merge it :-)

File: docs/installation.rst (hunk 2 above)
Comment (florian-huber) [reply docs/installation.rst]: Great, thanks!

**Pull request merged**